slash/react/src/List/ContentItemMono/ContentItemMono.tsx

```diff
@@ -0,0 +1,75 @@
+import { useMemo, type ReactNode } from "react";
+import { getComponentClassName } from "../../Form/core";
+import { ContentItemMonoSize } from "./constants";
+
+type ContentItemMonoProps = {
+  children: ReactNode;
+  className?: string;
+  classModifier?: string;
+  secondaryText?: string;
+  tertiaryText?: string;
+  leftElement?: ReactNode;
+  size?: ContentItemMonoSize;
+  isDisabled?: boolean;
+  isLeftElementCentered?: boolean;
+  hasStick?: boolean;
+};
+
+export const ContentItemMono = ({
+  children,
+  className,
+  classModifier,
+  secondaryText,
+  tertiaryText,
+  leftElement,
+  size = ContentItemMonoSize.M,
+  isDisabled = false,
+  isLeftElementCentered = false,
+  hasStick = false,
+}: ContentItemMonoProps) => {
+  const componentClassName = useMemo(() => {
+    let newClassModifier = classModifier ?? "";
+
+    if (isDisabled) {
+      newClassModifier += " disabled";
+    }
+
+    newClassModifier += ` ${size}`;
+
+    return getComponentClassName(
+      className,
+      newClassModifier,
+      "af-content-item-mono",
+    );
+  }, [classModifier, isDisabled, size, className]);
+
+  return (
+    <div className={componentClassName}>
+      {hasStick && (
+        <div className="af-content-item-mono__stick" role="presentation" />
+      )}
+      {leftElement && (
+        <div
+          className={`af-content-item-mono__left-container${
+            isLeftElementCentered
+              ? " af-content-item-mono__left-container--center"
+              : ""
+          }`}
+        >
+          {leftElement}
+        </div>
+      )}
+      <div className="af-content-item-mono__text-container">
+        <p className="af-content-item-mono__main-text"> {children}</p>
```

Comment:
```suggestion
        <p className="af-content-item-mono__main-text">{children}</p>
```

Reply:
Resolved in https://github.com/AxaFrance/design-system/pull/436/commits/44c0220f4a1e1f6482101600bd05f3cd697ff7fb